.env.example

```diff
@@ -0,0 +1,52 @@
+APP_NAME=Escooters
+APP_ENV=local
+APP_KEY=
+APP_DEBUG=true
+APP_URL=http://escooters.blumilk.localhost
+
+LOG_CHANNEL=stack
+LOG_DEPRECATIONS_CHANNEL=null
+LOG_LEVEL=debug
+
+DB_CONNECTION=pgsql
+DB_HOST=escooters-db-dev
+DB_PORT=5432
+DB_DATABASE=escooters
+DB_USERNAME=escooters
+DB_PASSWORD=password
+
+BROADCAST_DRIVER=log
+CACHE_DRIVER=file
+FILESYSTEM_DISK=local
+QUEUE_CONNECTION=sync
+SESSION_DRIVER=file
+SESSION_LIFETIME=120
```

Comment:
I think redis can be used for cache, queue, session

Reply:
changed